tests/definitions/run-command.sh

```diff
@@ -4,36 +4,40 @@ create-repo debian ${TESTREPO} > /dev/null 2>&1
 cd ${TESTREPO}
 repo_path=$(pwd)
 
-test_sequence() {
-  name="Run command in container"
-  pass "run rm --help | grep 'Remove (unlink) the FILE(s).'"
-
-  name="Run target from outside workspace"
-  pass "run workspace/target.sh | grep boom"
-
-  cd "${repo_path}/workspace"
-  name="Run target from inside workspace"
-  pass "run ./target.sh | grep boom"
-
-  cd /tmp
-  name="Run target outside repo - single command"
-  pass "run ${repo_path}/workspace/target.sh | grep boom"
-
-  name="Run target outside repo - split command"
-  pass "run ${repo_path} /workspace/target.sh | grep boom"
-}
-
 # Standard
 make > /dev/null 2>&1
 printf "#!/usr/bin/env sh\necho boom\n" > workspace/target.sh
 chmod +x workspace/target.sh
 subheading "Development builds"
-test_sequence
+
+
+name="Run command in container"
+pass "run rm --help | grep 'Remove (unlink) the FILE(s).'"
+
+name="Run target from repository root"
+pass "run workspace/target.sh | grep boom"
+
+cd "${repo_path}/workspace"
+name="Run target from inside workspace"
+pass "run ./target.sh | grep boom"
+
+cd /tmp
+name="Run target outside repository"
+pass "run --repo ${repo_path} workspace/target.sh | grep boom"
 
 
 # Build production image with target script
 cd ${repo_path}
 make production > /dev/null 2>&1
-rm workspace/target.sh
+#rm workspace/target.sh
```

Comment:
Is this still needed?

Reply:
Definitely - good spotting! The target should exist inside the production image so I wanted to remove it from the host to make sure it's not executing the host's copy.